packages/visx-demo/src/sandboxes/visx-brush/Example.tsx

```diff
@@ -194,6 +195,9 @@ function BrushChart({
             onClick={() => setFilteredStock(stock)}
             selectedBoxStyle={selectedBrushStyle}
             useWindowMoveEvents
+            renderBrushHandles={(props) =>
+              props.isBrushActive ? <BrushHandle {...props} /> : null
```

Comment:
it might make sense to _always_ render the `BrushHandle` here, and then let it handle rendering `null` or not